package.json

```diff
@@ -44,7 +44,7 @@
     "test:e2e:extension": "npm run test:e2e:build && npm run test:e2e:extension:run",
     "test:e2e:extension:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/specs/extension-installation.spec.ts",
     "test:e2e:pw": "npm run test:e2e:build && npm run test:e2e:pw:run",
-    "test:e2e:pw:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/specs/podman-machine-tests.spec.ts",
+    "test:e2e:pw:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/specs/kind.spec.ts",
```

Comment:
what is the intended purpose of this target?
the `pw` is very non-descriptive for me, and you completely changed the test it executed originally

Reply:
this is just for debugging, the original test was also added by me